src/framework/varQuery.mli

```diff
@@ -1,3 +1,5 @@
+(** Queries for constraint variables related to semantic elements. *)
```

Comment:
What do you mean by `semantic elements`?

Reply:
Since these are for manual restarting, they're things which the user could reference in a program, e.g. variables, types, program points, as opposed to some Goblint-internal things.